spec/javascripts/support/jasmine_helper.rb

```diff
@@ -0,0 +1,16 @@
+# Use this file to set/override Jasmine configuration options
+# You can remove it if you don't need it.
+# This file is loaded *after* jasmine.yml is interpreted.
+#
+# Example: using a different boot file.
+# Jasmine.configure do |config|
+#    config.boot_dir = '/absolute/path/to/boot_dir'
+#    config.boot_files = lambda { ['/absolute/path/to/boot_dir/file.js'] }
+# end
+#
+# Example: prevent PhantomJS auto install, uses PhantomJS already on your path.
+Jasmine.configure do |config|
+  if ENV['TRAVIS']
+    config.prevent_phantom_js_auto_install = true
```

Comment:
Why is this limited to `if ENV['TRAVIS']` and not applied all the time?

Other apps have this config set all the time:
eg https://github.com/alphagov/govuk_admin_template/blob/master/spec/javascripts/support/jasmine_helper.rb